autograder/autograder/grade_item.py

```diff
@@ -252,6 +252,7 @@ def grade_from_zip(my_autograding_zip_file,my_submission_zip_file,which_untruste
     checkout_path = os.path.join(tmp_submission, "checkout")
 
     provided_code_path = os.path.join(tmp_autograding, "provided_code")
+    solution_path = os.path.join(tmp_autograding, "solution")
```

Comment:
let's rename this folder instructor_solution
(So it's more obvious who's solution it is)

Reply:
@bmcutler Can you review it again?